src/components/ui/Suggest.jsx

```diff
@@ -159,3 +157,12 @@ Suggest.propTypes = {
 };
 
 export default Suggest;
+
+// Blur the focused field and close the suggest list when clicking outside of a field
+if (!window.onclick) {
+  window.onclick = e => {
+    if (e.target.tagName !== 'INPUT') {
+      fire('close_suggest');
```

Comment:
we might need to use the locally defined `close` function which will clear items ? I'm not sure

Reply:
yes, the event listener calls close()

Reply:
do we really need to register a global `close_suggest` event here ? I think all the logic can be kept inside `Suggest.jsx`